docs/user/whats-new.asciidoc

```diff
@@ -39,6 +39,87 @@ image::images/case-notification.png[Example email with case name, description, t
 
 To learn more, check out cases in the {observability-guide}/create-cases.html[{observability}], <<cases,{stack-manage-app}>>, or {security-guide}/cases-overview.html[{security-app}].
 
+[discrete]
+=== Dashboard
+
+[discrete]
+==== Wildcard search in dashboard controls
+Dashboard now supports searching for strings that are found within the values shown in a control. Previously, it was required for the string to be located at the start of the value in order to be found. However this is a more expensive option then just searching for the prefix, so it’s disabled by default. To enable it in Dashboard select Edit control > Options list settings > Contains. 
+
+[role="screenshot"]
+image::images/wildcard-search.png[Example of new string search]
+
+[discrete]
+=== Lens
+
+[discrete]
+==== Stacked bar chart legend
+
+The order of stacked bar chart legends is now based on popularity with the most important element displayed on top. You no longer need to scroll to the bottom of the legend to find the most popular data in the chart. 
+
+Previously:
+[role="screenshot"]
+image::images/oldwayhighlighted.png[Example of the old ordering of the chart legend, width=80%]
 
+New in 8.9:
+[role="screenshot"]
+image::images/newwayhighlighted.png[Example of the new ordering of the chart legend, width=80%]
+
+[discrete]
+==== Step curve
+
+Lens now supports three new options for line charts. The three options are straight, smooth, and step. Step curve line charts help users see clearly when data changes at irregular intervals, which helps to avoid misinterpretation. When creating visualizations select Line in the visualization type dropdown. Then use the visual options menu to choose between straight, smooth, or step in the Line interpolation dropdown. 
```

Comment:
```suggestion
**Lens** now supports three new options for line charts. The three options are straight, smooth, and step. Step curve line charts help users see clearly when data changes at irregular intervals, which helps to avoid misinterpretation. When creating visualizations select **Line** in the visualization type dropdown. Then use the visual options menu to choose between straight, smooth, or step in the Line interpolation dropdown. 
```